procServUtils/manage.py

```diff
@@ -207,11 +222,18 @@ def delproc(conf, args):
     stopproc(conf, args)
 
     _log.info("Disabling service procserv-%s.service", args.name)
-    SP.check_call([systemctl,
+    SP.call([systemctl,
```

Comment:
Why change to the un-checked call?  Have you seen this step fail?  As a general rule I don't like the idea of ignoring failures.

Reply:
I remember seeing this fail, but I cannot reproduce the problem now (it was months ago), so for the moment I will revert back to the ``check_call`` method.